src/test/lockDistribution.t.sol

```diff
@@ -120,7 +122,7 @@ contract LockDistributionTest is DSTest {
     // -------------------------------
 
     function testLockDistr() public { 
-        cheats.startPrank(eoa1);//******************** */
+        cheats.startPrank(eoa1, eoa1);//******************** */
```

Comment:
this was the problem. basically the single-argument function signature for `startPrank` does not set tx.origin at all, and so tx.origin is probably the testrunner that foundry uses or something weird, you know? there is a second function signature that takes two arguments, where the second argument sets `tx.origin` until you call `stopPrank`. that was the problem

https://book.getfoundry.sh/cheatcodes/start-prank?highlight=tx.origin#description